src/posix.mak

```diff
@@ -125,6 +126,11 @@ CFLAGS := $(WARNINGS) \
 	-fno-exceptions -fno-rtti \
 	-D__pascal= -DMARS=1 -DTARGET_$(OS_UPCASE)=1 -DDM_TARGET_CPU_$(TARGET_CPU)=1 \
 	$(MODEL_FLAG)
+# GCC Specific
+ifeq ($(HOST_CC), g++)
+CFLAGS := $(CFLAGS) \
+    -std=gnu++98
+endif
```

Comment:
hmm I don't get it. It looks in line with current [HEAD of master](https://github.com/D-Programming-Language/dmd/blob/master/src/posix.mak#L129)